.ci/vfio_jenkins_job_build.sh

```diff
@@ -249,45 +249,20 @@ install_dependencies() {
 			# cloud image dependencies
 			deps=(xorriso curl qemu-utils openssh-client)
 
-			# QEMU dependencies
-			deps+=(libcap-dev libattr1-dev libcap-ng-dev librbd-dev gcc pkg-config libglib2.0-dev libpixman-1-dev psmisc ninja-build)
-
 			sudo apt-get update
-			sudo apt-get install -y ${deps[@]}
+			sudo apt-get install -y ${deps[@]} qemu-kvm
 			;;
 		fedora|centos|rhel)
 			# cloud image dependencies
 			deps=(xorriso curl qemu-img openssh)
 
-			# QEMU dependencies
-			deps+=(libcap-devel libattr-devel libcap-ng-devel librbd-devel gcc glib2-devel pixman-devel psmisc ninja-build)
-
-			sudo dnf install -y ${deps[@]}
+			sudo dnf install -y ${deps[@]} qemu-kvm
```

Comment:
should we change the path to `qemu-kvm` ? sorry I don't know how distro packages work

https://github.com/kata-containers/tests/blob/270df0df73438db91695365e9c26acc4fe5e9c0b/.ci/vfio_jenkins_job_build.sh#L233

Reply:
@devimc weirdly enough the qemu binary remains the same! (verified on both fedora and ubuntu)

Reply:
Hmm.  I'd double check with the downstream QEMU maintainers as the path on CentOS / RHEL / Fedora seems to be different between each other.  RHEL doesn't support QEMU being called directly, this its binary is under `/usr/libexec/`.

I wonder if we should be trying to run it in all systems ... or just lower the scope and support it only on, let's say, Ubuntu & Fedora.

Reply:
I think in practice it used only in ubuntu, I'm not sure why fedora/centos/rhel is there

Reply:
@fidencio I tested the installation with a clean Fedora container and that was the path

Reply:
For Fedora, for sure.  For RHEL I don't think so.  That's the reason I'd double check with maintainers.
Even our spec file for kata-containers explicitly points at `/usr/libexec/` for the QEMU path on RHEL.

Reply:
yes, with RHEL won't work, on the other hand we don't use or test it....

Reply:
> yes, with RHEL won't work, on the other hand we don't use or test it.... 

... yet - @wainersm any future plan to test VFIO on RHEL/centOS ?

Reply:
The `qemu-kvm` package is, I believe, just a wrapper which depends on the correct `qemu-system-*` package (at least in Fedora, and I think it's similar in Ubuntu).  It may also install some extra dependencies and possibly a wrapper `qemu-kvm` binary, but it doesn't fundamentally change the qemu binary.

That said some distros (e.g. RHEL) put the qemu binary in a different place.  For now, I think covering only Ubuntu and Fedora is plenty.